terraform/nginx/default.conf.template

```diff
@@ -3,12 +3,8 @@ server {
   listen       ${PORT};
   
 
-  location = / {
-    proxy_pass  ${STATIC_URL}/index.html;
-    proxy_redirect off;
-  }
-  location = /login {
-    proxy_pass  ${STATIC_URL}/login.html;
+  location /login {
+    proxy_pass  ${STATIC_URL}/login;
     proxy_redirect off;
   }
```

Comment:
I think this will cause problems in production. GCS doesn't handle these redirects the same way as Nextjs does. Maybe we could split the nginx config between dev/prod? Or fix the nextjs API redirects?

Reply:
Splitting the nginx config seems like it would cause less sideffects on production, probably will go with that then